src/index.ts

```diff
@@ -10,6 +11,11 @@ import type { Deployment } from './riffraff';
 import { manifest, riffraffPrefix } from './riffraff';
 import { S3Store, sync } from './s3';
 
+/**
+ * Amazon STS expects OIDC tokens with the `aud` (audience) field set to `sts.amazonaws.com`
+ */
+const GITHUB_OIDC_AUDIENCE = 'sts.amazonaws.com';
```

Comment:
[This value is the default used as as input by the `configure-aws-credentials` action](https://github.com/aws-actions/configure-aws-credentials/blob/main/action.yml#L33-L36).